packages/toast/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@equinor/mad-toast",
+    "version": "1.0.0",
```

Comment:
Are you comfortable with major 1'ing this? Maybe we should keep it in 0 for some time to allow for more aggressive changes

Reply:
Lowered the version number so we have space to squash some bugs. But I hope to major upgrade the package as soon as possible, since I believe this will be our most stable package